lib/reversetunnel/remotesite.go

```diff
@@ -828,6 +828,13 @@ func (s *remoteSite) dialAndForward(params reversetunnelclient.DialParams) (_ ne
 		return nil, trace.Wrap(err)
 	}
 
+	// If the node is unknown (was directly dialed) tell the forwarding
```

Comment:
Does "unknown" mean "not in inventory?"

If so, can we rename usages of `isUnknownNode` to something like `isInventoryNode`?

Reply:
It does. But `isInventoryNode` seems reversed to me, what would make more sense is `isNotInventoryNode` as the node is *not* in the inventory

Reply:
Renamed to `isNotInventoryNode`